sources/Application/Instruments/WavFile.cpp

```diff
@@ -169,6 +175,11 @@ WavFile *WavFile::Open(const char *path) {
 	position+=wav->readBlock(position,4) ;
 	memcpy(&sampleRate,wav->readBuffer_,4) ;
 	sampleRate = Swap32(sampleRate);
+    #ifndef PARANOIDWAVFMTVALIDATION
```

Comment:
So we're forcing sample rate to always be 44100, disregarding its actual sample rate?
This will affect playback pitch for any samples with rates that didn't contain the sample rate in the expected block.

Reply:
Only if retrieved sampleRate is 0.